src/api/store/vendor.py

```diff
@@ -76,8 +76,7 @@ def create_vendor(self, context: Context, args) -> Tuple[Vendor, MassEnergizeAPI
 
       new_vendor = Vendor.objects.create(**args)
       if image:
-        logo = Media(name=f"Logo-{slugify(new_vendor.name)}", file=image)
-        logo.save()
+        logo = Media.objects.filter(pk = image[0]).first()
```

Comment:
are we asserting that at least one image is present? and that it is a list?  Also if its an array of id's maybe the word "image" is a tad bit misleading

Reply:
Yes, the array will never be empty, and will always have one item at least. In scenarios where we dont want to do anything with images, we do not provide the "image" field at all in the request payload. So this never triggers.